`npm test` does not succeed on fresh git clone (part two!)

Alternative title: I am this repo's CI solution

```
sam@hermes:~/SylverStudios/dunderlords$ npm test

> dunderlords@0.1.2 test /home/sam/SylverStudios/dunderlords
> elm-test


elm-test 0.19.1-revision2
-------------------------

Running 7 tests. To reproduce these results, run: elm-test --fuzz 100 --seed 297292015299988 /home/sam/SylverStudios/dunderlords/tests/CounterTest.elm /home/sam/SylverStudios/dunderlords/tests/HeroTest.elm /home/sam/SylverStudios/dunderlords/tests/MainTest.elm

↓ MainTest
↓ Testing view function
✗ Button has the expected text

    ▼ Query.fromHtml
    
        <div class="container">
            <header>
                <h1>
                    Dunderlords Alliance Builder
                </h1>
            </header>
            <section>
                <header>
                    <h2>
                        Team
                    </h2>
                </header>
                <div class="crew-empty">
                    Click a Hero below to begin building a team
                </div>
            </section>
            <section>
                <h2>
                    Hero Pool
                </h2>
                <div class="pool">
                    <div>
                        <img class="hero" alt="Earth Spirit Mini Badge" src="/images/minis/earth-spirit.png">
                        Earth Spirit
                    </div>
                    <div>
                        <img class="hero" alt="Juggernaut Mini Badge" src="/images/minis/juggernaut.png">
                        Juggernaut
                    </div>
                    <div>
                        <img class="hero" alt="Pudge Mini Badge" src="/images/minis/pudge.png">
                        Pudge
                    </div>
                    <div>
                        <img class="hero" alt="Slardar Mini Badge" src="/images/minis/slardar.png">
                        Slardar
                    </div>
                    <div>
                        <img class="hero" alt="Tidehunter Mini Badge" src="/images/minis/tidehunter.png">
                        Tidehunter
                    </div>
                    <div>
                        <img class="hero" alt="Tiny Mini Badge" src="/images/minis/tiny.png">
                        Tiny
                    </div>
                    <div>
                        <img class="hero" alt="Troll Warlord Mini Badge" src="/images/minis/troll-warlord.png">
                        Troll Warlord
                    </div>
                    <div>
                        <img class="hero" alt="Tusk Mini Badge" src="/images/minis/tusk.png">
                        Tusk
                    </div>
                </div>
            </section>
        </div>
    
    
    ▼ Query.findAll [ tag "button" ]
    
    0 matches found for this query.
    
    
    ▼ Query.first
    
    0 matches found for this query.
    
    
    ✗ Query.first always expects to find 1 element, but it found 0 instead.



TEST RUN FAILED

Duration: 133 ms
Passed:   6
Failed:   1

npm ERR! Test failed.  See above for more details.
```

--- a/.github/workflows/elm-ci.yml
+++ b/.github/workflows/elm-ci.yml
@@ -1,0 +1,19 @@
+name: Elm CI
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
+    - name: Install NPM packages
+      run: npm install
+    - name: Run tests
+      run: npm run test
+      env:
+        CI: true

